File: ios/ExpoMqttBroker.m
#import <React/RCTBridgeModule.h>

@interface RCT_EXTERN_MODULE(ExpoMoquette, NSObject)

RCT_EXTERN_METHOD(startServerAsync:(float)a withB:(float)b
                 withResolver:(RCTPromiseResolveBlock)resolve
                 withRejecter:(RCTPromiseRejectBlock)reject)

@end

File: ios/ExpoMqttBroker.swift
@objc(ExpoMoquette)
class ExpoMoquette: NSObject {

    @objc(startServerAsync:withB:withResolver:withRejecter:)
    func startServerAsync(a: Float, b: Float, resolve:RCTPromiseResolveBlock,reject:RCTPromiseRejectBlock) -> Void {
        resolve(a*b)
    }
}
